project_task_analytic_propagation/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2024 Moduon Team S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/LGPL-3.0)
+
+{
+    "name": "Project Task Analytic Propagation",
+    "summary": "Updates timesheet's analytic account"
+    " when their task changes the analytic.",
+    "version": "18.0.1.0.0",
+    "development_status": "Alpha",
+    "category": "Timesheet",
+    "website": "https://github.com/OCA/timesheet",
+    "author": "Moduon, Odoo Community Association (OCA)",
+    "maintainers": ["edlopen", "rafaelbn"],
```

Comment:
You can remove @edlopen if he doesn't have a problem. AFAIK he's no longer maintaining this. Then add yourself:


```suggestion
    "maintainers": ["Andrii9090", "rafaelbn"],
```

> [!Note]
> If you do this in further migrations, remember to always ping the former maintainer. It's good etiquette.